.env.sample

```diff
@@ -6,4 +6,5 @@ NEXT_PUBLIC_DISPLAY_DENOM_EXPONENT=6
 NEXT_PUBLIC_GAS_PRICE=0.03uatom
 NEXT_PUBLIC_CHAIN_ID=cosmoshub-4
 NEXT_PUBLIC_ADDRESS_PREFIX=cosmos
-NEXT_PUBLIC_EXPLORER_LINK_TX=https://www.mintscan.io/cosmos/txs/%s
+NEXT_PUBLIC_EXPLORER_LINK_TX="https://www.mintscan.io/cosmos/txs/${txHash}"
```

Comment:
I wanted to backport this to latest master. Turns put that `${txHash}` is evaluated to "" when `NEXT_PUBLIC_EXPLORER_LINK_TX` is read. Then `process.env.NEXT_PUBLIC_EXPLORER_LINK_TX` becomes `https://www.mintscan.io/cosmos/txs/`.

I can also inject environment data here:

```
txHash="HACKME" npm run dev

# and


console.log(process.env.NEXT_PUBLIC_EXPLORER_LINK_TX);
```

gives me `https://testnet.tgrade.aneka.io/txs/HACKME`. If I use

```
NEXT_PUBLIC_EXPLORER_LINK_TX="https://www.mintscan.io/cosmos/txs/\${txHash}"
```

it works as expected.

This feels a bit scary to me. We can either require the escaping here or accept both:


```js
  if (process.env.NEXT_PUBLIC_EXPLORER_LINK_TX) {
    return process.env.NEXT_PUBLIC_EXPLORER_LINK_TX.replace("${txHash}", hash).replace("%s", hash);
  }
```

Any thoughts?

Reply:
ah good catch! I prefer doing the `\` escape in the .env file, it feels simpler and more consistent. added in 12f94a21c501e3e43eb38b060aa5025654280e01